modules/ensembl/annotation/dumpanno/meta.yml

```
@@ -0,0 +1,48 @@
+---
+name: "annotation_dumpanno"
+
+description: This module is used to dump the functional annotation of a genome.
+
+keywords:
+  - core_database
+  - ensembl-legacy-scripts
+  - fasta
+  - docker
+  - container
+  
+tools:
+  - "annotation":
+      homepage: "https://github.com/Ensembl/nextflow_modules"
+      description: "Module which function to dump annotation directly from a core database."
+      license: "Apache License version 2.0"
+
+input:
+  # Only when we have meta
+  - - meta:
+        type: map
+        description: "A meta map including 'core database information'."
+          e.g. `[ id:'db_name', server_details:[host : 'db'] ]`
+          pattern: "[a-z]+_[a-z]_gc[af][0-9]v[0-9]+_core_*"
```

There is something off in this code, but not sure what it may be. Maybe should look like this?
```suggestion
        description: |
          A meta map including 'core database information'.
          e.g. [ id:'db_name', server_details:[ host:'db' ] ]
        pattern: "[a-z]+_[a-z]_gc[af][0-9]v[0-9]+_core_*"
```